library/core/src/ptr/const_ptr.rs

```diff
@@ -66,6 +66,32 @@ impl<T: ?Sized> *const T {
         self as _
     }
 
+    /// Try to cast to a pointer of another type by checking aligment.
+    ///
+    /// If the pointer is well aligned to the target type, it will be casted
```

Comment:
Looking at the docs it seems we use "properly aligned" rather than "well aligned", so it would be good to match that here.